plugins/MauticEmailMarketingBundle/Config/config.php

```diff
@@ -19,17 +19,17 @@
         'forms' => [
             'mautic.form.type.emailmarketing.mailchimp' => [
                 'class'     => 'MauticPlugin\MauticEmailMarketingBundle\Form\Type\MailchimpType',
-                'arguments' => ['mautic.factory', 'session', 'mautic.helper.core_parameters'],
+                'arguments' => ['mautic.helper.integration', 'mautic.plugin.model.plugin', 'mautic.helper.core_parameters'],
                 'alias'     => 'emailmarketing_mailchimp',
```

Comment:
Remove also the aliases for form types from the config.

Reply:
Fixed in c00fd4ab797afa58513301ee7fbe59568c67abeb